HandyHansel/Attributes/BotCategoryAttribute.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace HandyHansel.Attributes
+{
+    public class BotCategoryAttribute : Attribute
+    {
+        public string Name { get; set; }
+
+        public BotCategoryAttribute(string name)
+        {
+            this.Name = name;
+        }
+    }
+}
```

Comment:
Adds an attribute that allows you to create categories of commands for use in the HelpFormatter